BEYOND_BITSWAP/README.md

```diff
@@ -15,49 +15,57 @@ In high level, this project is about:
 In short, the aim of the project is two-fold: to drive speed-ups in file-sharing for IPFS and other P2P networks; and to enable a framework for anyone to join the quest of designing, implementing and evaluating brand new file-sharing strategies in P2P networks. 
 
 ## Why the project code name?
+
 Bitswap has been for some time the file-sharing subsystem within IPFS, then Graphsync came to propose a new way of approaching file-sharing on IPFS. The scope of the project is not only to improve Bitswap's performance, but file-sharing in P2P networks as a whole. We don't restrict ourselves exlusively to Bitswap or IPFS for our exploration.
 
 Being said that, the fact that IPFS had an infrastructure in place to start testing our ideas, and Bitswap being its file-sharing module, made us start our initial explorations over Bitswap and IPFS, but our aim is to go way farther and improve file-sharing performance with new protocols and proposals every P2P network can leverage and benefit from. In short, we want to go "Beyond Bitswap". The project can be considered a success if by the end of it one has a set of pluggable protocols and modules to achieve file-sharing in P2P environments, along with all the testbeds, tools and benchmarks required to improve this protocols and go _"Beyond Bitswap"_.
 
+## 💌 Invite to Research with us
+
+ResNetLab collabores with over 10 Research Groups all over the world and Protocol Labs Research has developed research collaborations in multiples of ten in the last few years. We are always eager to collaborate with more researchers in all kinds of capacity, from thesis project (M.Sc or PhD), to Post-Doc, Grants, RFPs and independent research projects.
+
+We are making all our contributions, ideias, testbed, benchmarking and analysis scripts below. You are more than welcome to pick on these and drive one, or build on top. If you have questions, please [mail us](mailto:resnetlab@protocol.ai).
+
 ## Contributions & Results
 
 ### Documents
```

Comment:
@adlrocha can you make sure that the docs shared have the following perms:

![image](https://user-images.githubusercontent.com/1211152/99244371-e3f8b780-27f9-11eb-8754-e7194209ccfb.png)